### PR TITLE
feat(common): replace nginx

### PR DESCRIPTION
**Pull request by McDaddy**
## What this PR does / why we need it:
replace nginx web service by nestjs node server

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: scheduler/package.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
   "private": true,
   "scripts": {
     "start": "nest start --watch",
+    "build": "rm -rf ./dist && nest build",
+    "start:prod": "node dist/main",
```
Comment (daskyrk) [inline scheduler/package.json:10]: Suggest to use pm2 to protect the node process.

File: scheduler/src/proxy.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+export const createProxyService = (app: INestApplication) => {
+  app.use(
+    `/api/**/${dataAppName}-websocket`,
+    createProxyMiddleware({
+      target: API_URL,
+      ws: true,
+      changeOrigin: !isProd,
+      pathRewrite: replaceApiOrgPath,
+      onProxyReqWs: (proxyReq, req: Request, socket) => {
+        isProd && proxyReq.setHeader('org', extractOrg(req.headers.referer));
```
Comment (daskyrk) [inline scheduler/src/proxy.ts:43]: Should always set org header no matter is or not prod?

File: scheduler/package.json (hunk 1 above)
Comment (McDaddy) [reply scheduler/package.json:10]: can't resolve how to get pm2 log by kubectl logs command for now.  leave it to the next step

**Pull request merged**